argus/debug/debuger.py

```diff
@@ -0,0 +1,124 @@
+# Copyright 2014 Cloudbase Solutions Srl
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+from multiprocessing.connection import Client
+
+class EmptyDebuger():
+    """Template class for null debugger.
+
+    Everything that is implemented here is null so that debugger calls around
+    the code won't need to be removed in case argus is executed in normal 
+    mode. Every method present in other debuggers should be written here as an
+    empty one.
+    """
+
+    def __init__(self):
+        pass
+
+    def pause(self, *args):
+        pass
+
+    def run_python(self, *args):
+        pass
+
+    def wait(self, func):
+        def _inner(*args, **kwargs):
```

Comment:
It's not useless because if anywhere around the code exists a decorator like this and debug flag it's false then an error will appear.